src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.html

```diff
@@ -27,7 +27,7 @@
           <tr>
             <td i18n
                 class="bold">System</td>
-            <td>{{ user.system | booleanText }}</td>
+            <td>{{ user.system === 'true' | booleanText }}</td>
```

Comment:
Instead of fixing this specific case here, I'd either fix it in:
* Data coming from the back-end
* `booleanText` pipe.

Reply:
> Instead of fixing this specific case here, I'd either fix it in:
> 
>     * Data coming from the back-end
> 
>     * `booleanText` pipe.

I considered both before, but:
1) BooleanText is meant to handle booleans (for handle boolean-like strings there is ```BooleanPipe```).
2) Injecting BooleanPipe in component so the transformation happens right after the payload seemed an overkill to me so I preferred to follow KISS principle for the sake of smooth backporting.

I think we can agree on this:
```
user.system | boolean | booleanText
```